### DISCUSSION TITLE
Using CTRL+TAB to navigate to open code tabs

### DISCUSSION THREAD

**Discussion by nikneem** [community/community#3646] [Codespaces]
Like in VS Code, it would be awesome if I could use the CTRL+TAB key combination to navigate through opens code tabs. New, this toggles my browser tabs which is a really confusing experience.

**Comment by bailey** [comment]
Hey there! 

In the browser, we can't override all of the existing key bindings. You do have options, though! 

- You can download the PWA for your codespace, which allows you to use the key bindings that you're used to in VS Code. 
- Connect to your codespace directly from your local installation of VS Code using the GitHub Codespaces extension. 

Let me know how it goes! ✨

**Reply by nikneem** [reply]
Hi @bailey,
Thanks for your answer, I'm trying to connect my 'local' VS Code to my codespace but it fails to connect. I'm probably doing something wrong but I can't figure out what. I'm testing with my Azure Functions JWT Validation input binding project right here: https://github.com/nikneem/azure-functions-jwt-binding

As you can see I added a dev container and when I spin it up using 'online' codespaces I click the menu all the way in the upper-left corner and click 'Open in VS Code'. Then VS Code starts and I see a progress bar for about a minute or three before I get an error message. Please note I'm new to codespaces as Martin gave me access two days ago during your presentation at the Summit.

**Reply by bailey** [reply]
Hey! I'm sorry to hear you're having problems. I'll loop in @2percentsilk in case there are any known problems with the 'Open in VS Code' feature. 

In the meantime, you can actually install the [GitHub Codespaces extension](https://code.visualstudio.com/docs/remote/codespaces) and create new or connect to existing codespaces from within the extension. Let me know if that works for you 👀
